apps/wpcom-block-editor/src/wpcom/features/fix-block-invalidation-errors.js

```diff
@@ -42,13 +43,13 @@ async function fixInvalidBlocks() {
 	}
 
 	// If any blocks have validation issues auto-fix them for now, until core is less strict.
-	select( 'core/editor' )
+	select( blockEditorStore )
```

Comment:
We seem to be referencing the block editor here, when originally it was referencing the editor store.